Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest-tasks-navigation/ContestTasksNavigation.tsx

```diff
@@ -70,6 +72,8 @@ const ContestTasksNavigation = () => {
                       type={ButtonType.plain}
                     >
                         {problem.name}
+                        {problem?.isExcludedFromHomework &&
```

Comment:
As per @dinikolaeva's comment (https://github.com/SoftUni-Internal/exam-systems-issues/issues/890#issuecomment-1702352909) , editing Problem Group type from Problem Edit should update the problem group type. Currently that value is not persisted.